libfdcore/fdd.y

```diff
@@ -665,3 +669,31 @@ tls_dh:			TLS_DH_BITS '=' INTEGER ';'
 				fclose(fd);
 			}
 			;
+			
+untrusted_avp_vendors:			UNTRUSTED_AVP_VENDORS '=' vendor_list ';'
+			{
+				conf->cnf_untrusted_avp_vendors = $3;
+			}
+			;
+			
+vendor_list:			vendor_list ',' INTEGER
+			{
+				uint32_t* arr = $1;
+				int len = 0;
+				while (arr[len] != 0) { ++len; };
+				$$ = (uint32_t*)realloc(arr, sizeof(uint32_t) * (len + 2));
+				$$[len] = $3;
+				$$[len + 1] = 0;
+			}
+			| INTEGER
+			{
+				$$ = (uint32_t*)calloc(2, sizeof(uint32_t));
+				$$[0] = $1;
+				$$[1] = 0;
+			}
+			|
```

Comment:
I think elsewhere they use `/* empty */` to make it clear this isn't just missed out code.